### ISSUE TITLE
The accuracy of the model

### ISSUE THREAD

**Issue by famishedfish** [aioz-ai/MICCAI19-MedVQA#13]
Dear author,
I followed the training step strictly and used the data_RAD/ file from the link you offered. But I can only get around 59% accuracy when testing BAN_MEVF model. 
I could reproduce the accuracy of 62% using the pretrained BAN_MEVF model.
I wonder if it’s because the pretrained ae model or maml model in data_RAD/ is not the best?

**Comment by xuanbinh-nguyen96** [comment]
Dear Xiaoyu Zhang,
We have checked both mentioned models and have not found any problem with them. We still get similar testing performance after training again. Our train log file can be found at: [link_log_BAN_MEVF](https://vision.aioz.io/f/eacad68f5e264efa8aa3/?dl=1)
However, we have figured out that random weights generation before training may affect the training scheme. Currently, we use the initial seed (--seed argument in main.py) to stabilize the training process within our hardware setup but can not make sure if there are differences between our hardware and your ones.
You can try changing the mentioned initial seed and try to find a better one for your hardware.
Sincerely!

**Comment by famishedfish** [comment]
Thanks for the detailed reply!
I trained this model on RTX 3090, and changing the initial seed does make the accuracy higher.
Though I haven't got the highest performance, I guess I could try more times.
Sincerely!
